lib/features/home_view/widgets/news_section.dart

```diff
@@ -23,9 +24,7 @@ class NewsSection extends ConsumerWidget {
           SubsectionHeader(
               title: context.localize.whats_up,
               onClick: () {
-                ref
-                    .read(bottomNavBarControllerProvider.notifier)
-                    .goTo(NavBarEnum.info);
+                ref.read(navigatorProvider).changeTabBar(NavBarEnum.info);
```

Comment:
And here as well, maybe it's worth refactoring. Just to be consistent.

Reply:
okey. it's defienetly more consistent like this